cassandra-analytics-core/src/main/java/org/apache/cassandra/spark/bulkwriter/RecordWriter.java

```diff
@@ -372,19 +372,33 @@ private void writeRow(Tuple2<DecoratedKey, Object[]> keyAndRowData,
     private Map<String, Object> getBindValuesForColumns(Map<String, Object> map, String[] columnNames, Object[] values)
     {
         Preconditions.checkArgument(values.length == columnNames.length,
-                                    "Number of values does not match the number of columns " + values.length + ", " + columnNames.length);
+                "Number of values does not match the number of columns " + values.length + ", " + columnNames.length);
```

Comment:
It is surprising if checkstyle is happy with this change. Basically, the parameters should align with the previous line. If the string is too long, you can break the string into multi-line. 
For this particular change, it does not seem to be relevant to the patch. Please revert.